sdk/loadtestservice/azure-developer-loadtest/setup.py

```diff
@@ -0,0 +1,86 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+# coding: utf-8
+
+import os
+import re
+from setuptools import setup, find_packages
+
+
+
+PACKAGE_NAME = "azure-developer-loadtesting"
+PACKAGE_PPRINT_NAME = "Azure Developer LoadTesting"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace("-", "/")
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, "_version.py"), "r") as fd:
+    version = re.search(
+        r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE
+    ).group(1)
+
+if not version:
+    raise RuntimeError("Cannot find version information")
+
+
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description="Microsoft Azure Developer LoadTesting Client Library for Python",
+    
+    long_description=open("README.md", "r").read(),
+    long_description_content_type="text/markdown",
+    license="MIT License",
+    author="Microsoft Corporation",
+    
+    author_email="azpysdkhelp@microsoft.com",
+    url="https://github.com/Azure/azure-sdk-for-python/tree/main/sdk",
+    keywords="azure, azure sdk",
+    
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "License :: OSI Approved :: MIT License",
+    ],
+    zip_safe=False,
+    packages=find_packages(
+        exclude=[
+            "tests",
+            
+            # Exclude packages that will be covered by PEP420 or nspkg
+            
+            "azure",
+            "azure.developer",
+        ]
+    ),
+    include_package_data=True,
+    package_data={
+        'pytyped': ['py.typed'],
+    },
+    
+    install_requires=[
+        
+        "isodate>=0.6.0",
+        
+        
+        "azure-core<2.0.0,>=1.2.2",
+        
+    ],
+    
+    python_requires=">=3.6",
+    
+)
```

Comment:
The code format is in a mess. Please update code with `autorest --version=3.8.1 --use=@autorest/python@6.0.0 --use=@autorest/modelerfour@4.23.5 README.md`

Reply:
Cleaned setup.py in https://github.com/Azure/azure-sdk-for-python/pull/25264/commits/4a6d627c9abe708dd37ff12abbcd6ec8dbcbd20b

Reply:
`autorest --version=3.8.1 --use=@autorest/python@6.0.0 --use=@autorest/modelerfour@4.23.5 README.md` this cmd shall help to reformat the generated sdk code. There are many blank lines in `setup.py`: https://github.com/NiveditJain/azure-sdk-for-python/blob/f3039cb36e24adfa67710893c7f0b6564bd9a279/sdk/loadtestservice/azure-developer-loadtesting/setup.py#L78-L79